config/configuration.go

```diff
@@ -532,6 +532,10 @@ func CreateConfigFile(args map[string]interface{}) (*viper.Viper, error) {
 	txPoolAccess := args["txpoolaccess"].(bool)
 	preCommitEnabled := args["preCommitEnabled"].(bool)
 	apiHost := args["apiHost"].(string)
+	webhookURL, ok := args["webhookURL"].(string)
```

Comment:
I think we can simplify this to:
```suggestion
	webhookURL, _ := args["webhookURL"].(string)
```
or even without the `_` if we pass it always as the rest of the fields.